go.mod

```diff
@@ -5,11 +5,11 @@ go 1.18
 require (
 	cosmossdk.io/math v1.0.0-beta.3
 	github.com/armon/go-metrics v0.4.1
-	github.com/cosmos/cosmos-sdk v0.46.1
+	github.com/cosmos/cosmos-sdk v0.46.2-0.20220831122102-a95c62680975
```

Comment:
should we update to a stable version?

Reply:
no, this is what ethermint is using

Reply:
I think we can use 0.46.1

Reply:
there are changes in the main branch of the 46 sdk that make upgrades much faster.  When cro upgraded to 46, the archive nodes took 16 hours so we should use the latest commits there I think